Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "e310x-hal"
-version = "0.9.2"
+version = "0.9.3"
```

Comment:
Hmm, I think this change is breaking and requires a minor version bump.

Reply:
This is some sort of rebase issue, for some reason the commit from master gets shown up here. I'm not quite sure how that happened.

Reply:
force push fixed it.